code/modules/mob/living/simple_animal/borer.dm

```diff
@@ -163,7 +163,8 @@ var/global/borer_unlock_types_leg = typesof(/datum/unlockable/borer/leg) - /datu
 			if(health < 20)
 				health += 0.5
 			if(chemicals < 250 && !channeling)
-				chemicals++
+				var/new_chemicals = chemicals + 2
```

Comment:
Why bother with this var? Just use`chemicals + 2` in the next line

Reply:
So that it doesn't get to 251

Reply:
Oh I see your point